### PR TITLE
improve the description of import format option

### PR DESCRIPTION
**Pull request by renetapopova**
Depends on https://github.com/neo-technology/neo4j/pull/25631

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1638.surge.sh

Updated pages:
- [Import](https://neo4j-docs-operations-1638.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1638.surge.sh

Updated pages:
- [Manage roles](https://neo4j-docs-operations-1638.surge.sh/operations-manual/5/authentication-authorization/manage-roles)
- [Import](https://neo4j-docs-operations-1638.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)
- [Administering immutable privileges](https://neo4j-docs-operations-1638.surge.sh/operations-manual/5/tutorial/tutorial-immutable-privileges)

**Comment by renetapopova** [comment]
Approved in the codebase.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1638.surge.sh

Updated pages:
- [Manage roles](https://neo4j-docs-operations-1638.surge.sh/operations-manual/5/authentication-authorization/manage-roles)
- [Import](https://neo4j-docs-operations-1638.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)
- [Administering immutable privileges](https://neo4j-docs-operations-1638.surge.sh/operations-manual/5/tutorial/tutorial-immutable-privileges)

File: modules/ROOT/pages/tutorial/tutorial-immutable-privileges.adoc [hunk 1]
```diff
@@ -53,7 +53,6 @@ SHOW PRIVILEGES WHERE IMMUTABLE
 |"database"
 |"PUBLIC"
 |true
-6+a|Rows: 1
 |===
```
Comment (renetapopova) [inline modules/ROOT/pages/tutorial/tutorial-immutable-privileges.adoc:56]:
```suggestion

7+a|Rows: 1
|===
```

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1638.surge.sh

Updated pages:
- [Manage roles](https://neo4j-docs-operations-1638.surge.sh/operations-manual/5/authentication-authorization/manage-roles)
- [Import](https://neo4j-docs-operations-1638.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)
- [Administering immutable privileges](https://neo4j-docs-operations-1638.surge.sh/operations-manual/5/tutorial/tutorial-immutable-privileges)

**Pull request merged**